src/classdef.cpp

```diff
@@ -5045,9 +5045,9 @@ QCString ClassDefImpl::inheritanceGraphFileName() const
   return m_impl->inheritFileName;
 }
 
-void ClassDefImpl::setTypeInheritanceGraph(CLASS_GRAPH_t e)
+void ClassDefImpl::setTypeInheritanceGraph(CLASS_GRAPH_t e, bool explicitSet)
```

Comment:
Since this function only does something when `explicitSet` is `true` why call `setTypeInheritanceGraph` at all unless `explicitSet` is true (and then omit the parameter)?